packages/frontend/src/components/Search/FilterSearch.tsx

```diff
@@ -248,16 +289,16 @@ export const FilterSearch = (props: FilterSearchProps) => {
           <Select
             displayEmpty
             label={t('search:sort_results')}
-            value={sortType}
-            onChange={(event) => setSortType(event.target.value)}
+            value={sort}
+            onChange={(event) => updateSort(event.target.value as Sort)}
             input={<OutlinedInput />}
             sx={{ width: '100%' }}
           >
-            <MenuItem value={SortDataType.NONE}>{t('search:no_sort')}</MenuItem>
-            <MenuItem value={SortDataType.ASC_NAME}>{t('search:A-Z')}</MenuItem>
-            <MenuItem value={SortDataType.DESC_NAME}>{t('search:Z-A')}</MenuItem>
-            <MenuItem value={SortDataType.DESC_MODIFIED}>{t('search:latest_first')}</MenuItem>
-            <MenuItem value={SortDataType.ASC_MODIFIED}>{t('search:oldest_first')}</MenuItem>
+            <MenuItem value={SortDataType.NONE as any}>{t('search:no_sort')}</MenuItem>
```

Comment:
Instead of `any`, can you give these the type of `Sort` from `packages/frontend/src/types/search.ts`?